tidy3d/plugins/waveguide/rectangular_dielectric.py

```diff
@@ -989,6 +1005,7 @@ def plot_structures_eps(
             cbar=cbar,
             reverse=reverse,
             ax=ax,
+            transpose=transpose,
```

Comment:
If my memory is correct, `Scene.plot_structures_eps()` (invoked by this function) was tested visually (with `transpose=True`) using the [ThermallyTunedRingResonator.ipynb](https://github.com/flexcompute/tidy3d-notebooks/blob/develop/ThermallyTunedRingResonator.ipynb) notebook example.
You might want to verify it again, however.